spec/controllers/comments_controller_spec.rb

```diff
@@ -154,10 +154,9 @@
       context "when logged in as an admin" do
         before { fake_login_admin(create(:admin)) }
 
-        it "renders the :new template" do
+        it "asks to log out first" do
           post :new, params: { tag_id: fandom.name }
```

Comment:
I think we need to add a test for the `add_comment_reply` action as well. And can we tweak the description to better describe the behavior, i.e., `it "redirects to root with notice prompting log out"` or similar?